features/step_definitions/cross_compilation.rb

```diff
@@ -1,6 +1,6 @@
-# Naive way of looking into platforms, please include others like FreeBSD?
+# Naive way of looking into platforms
 Given %r{^I'm running a POSIX operating system$} do
-  unless RbConfig::CONFIG['host_os'] =~ /linux|darwin/ then
+  unless RbConfig::CONFIG['host_os'] =~ /linux|darwin|bsd|dragonfly/i then
```

Comment:
Honestly I'm not sure, on FreeBSD it is not needed:

    [2] pry(main)> RbConfig::CONFIG['host_os']
    => "freebsd10"

I think it has to do with DragonFly, I'm going to check and will
confirm in a few days.